packages/0xcert-ethereum-gateway-contracts/src/contracts/actions-gateway.sol

```diff
@@ -322,21 +313,17 @@ contract ActionsGateway is
       temp = keccak256(
```

Comment:
How do we protect against an order which is valid on one gateway being executed against another gateway on the same chain? (INCLUDING prior version of 0xcert proxies?)

How do we protect against an order which is valid on another chain (Wanchain, Ropsten, plasma) being replayed on Ethereum main net?

Reply:
All hashes include the address of the actionsGateway on which they are performed and we never deploy contracts in a way they would have the same contract address.

Reply:
This warning is available [here](https://github.com/0xcert/framework/tree/2.0/packages/0xcert-ethereum-gateway-contracts). @fulldecent good?

Reply:
Thank you, warning is good. Resolved.